### ISSUE TITLE
[Glimmer 2]{{debugger}}

### ISSUE THREAD

**Issue by mitchlloyd** [emberjs/ember.js#14214]
I miss it.

Poking around in the internal helpers and `syntax` classes I couldn't find any obvious way to get the same information that was available from the [HTMLbars debugger keyword](https://github.com/emberjs/ember.js/blob/v2.7.2/packages/ember-htmlbars/lib/keywords/debugger.js#L58-L60). Any ideas?

I was wondering whether this might be possible with an AST rewrite. So something like this:

```hbs
{{#each things as |thing|}}
  {{debugger}}
{{/each}}
```

could become 

```hbs
{{#each things as |thing|}}
  {{debugger this thing=thing}}
{{/each}}
```

Which would make writing the internals of the debugger helper pretty simple at that point.

**Comment by rwjblue** [comment]
@mitchlloyd - Great suggestion! We definitely need to get the `get('somePathHere')` functionality back, and allow access to `context`.

For others looking into it, [this](https://github.com/emberjs/ember.js/blob/v2.7.2/packages/ember-htmlbars/lib/keywords/debugger.js#L55-L64) is what we did in HTMLBars land.

I have been chatting with @mmun about it, and I think the solution for both may be to generate the spec statements manually.  I'd need to chat with @chancancode / @wycats to confirm.

I'm going to label this as a bug though, because reducing the debuggability of the template layer is definitely a bug to me...

**Comment by chancancode** [comment]
Fixed in #14231
